replies.go

```diff
@@ -144,7 +144,7 @@ var replies = []Reply{
 		message: "0) Schematic file **MUST** be made in a 1.12.2 world or prior. 1) Place the .schematic file into `.minecraft/schematics`. 2) Ensure all the blocks are in your hotbar. 3) Type `#build name.schematic`",
 	},
 	{
-		pattern: `((crack|cracked) (launcher|account|game|minecraft))|(terramining|shiginima|(t(-|)launcher))`,
+		pattern: `((crack|cracked) (launcher|account|game|minecraft))|(terramining|shiginima|(t(-|\s)launcher))`,
```

Comment:
I'd prefer something like `[-\s]+` rather than `(-|\s)`.

If we're going to match "**t launcher**" we should avoid matching things like (e.g.) "shi**t launcher**". The best way to do that would be with word boundaries `\b`, e.g. `\bt[-\s]*launchers?\b`